bundles/org.openhab.binding.darksky/src/main/java/org/openhab/binding/darksky/internal/handler/DarkSkyWeatherAndForecastHandler.java

```diff
@@ -136,9 +136,10 @@ public void initialize() {
         try {
             location = new PointType(config.location);
         } catch (IllegalArgumentException e) {
-            location = null;
+            logger.warn("Error parsing 'location' parameter: ", e.getMessage());
```

Comment:
I should add the placeholder here ...

```suggestion
            logger.warn("Error parsing 'location' parameter: {}", e.getMessage());
```